lib/src/processors/commons/abstract_file_string_processor.dart

```diff
@@ -38,6 +41,10 @@ abstract class AbstractFileStringProcessor extends AbstractFileProcessor {
 
   @override
   void execute() {
+    Directory directory = Directory(path_dart.dirname(path));
+    if (!directory.existsSync()) {
+      directory.createSync(recursive: true);
+    }
```

Comment:
This is the solution I was mentioning before:
https://github.com/AngeloAvv/flutter_flavorizr/pull/219/files